pallets/schema/src/tests.rs

```diff
@@ -314,3 +314,48 @@ fn test_schema_lookup() {
 		}
 	});
 }
+
+#[cfg(test)]
```

Comment:
It is better to extend the existing framework with new tests than create a mod. The #[cfg(test)] is activated with #[test] macro. Also, rather than passing a custom MAX_SCHEMA_SIZE, use the MaxEncodedSchemaLength value from mock.

Reply:
[Change made according to the review](https://github.com/dhiway/cord/pull/435/commits/3387095bd40868e0a41ae829290f22206c8822e4) Sir i have made changes according to the review.